docs/src/appendix/sncast/account/deploy.md

```diff
@@ -14,3 +14,23 @@ Name of the (previously created) account to be deployed.
 Optional.
 
 Maximum fee for the `deploy_account` transaction. When not used, defaults to auto-estimation.
+
+## `--fee-token <FEE_TOKEN>`
```

Comment:
I think those deserve their own guide on how to use it, so it's not confusing for the users